templates/dataset-detail.php

```diff
@@ -195,3 +195,9 @@ function render_metadata_table($supported_fields,$data){
 }
 
 ?>
+<script>
+  //Send GA Event
+  jQuery(function(){
+    ga('send', 'event', 'Dataset', 'view', '<?php echo $dataset_id; ?>');
```

Comment:
@DragonVirus I do not see the GA library being added anywhere? Is this something you forgot and you might still need to add via bower or composer? Or is it already added via the Google Analytics dashboard plugin already installed?